acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -134,7 +134,7 @@ def ruby_command(host)
       def gem_command(host, type='aio')
         if type == 'aio'
           if host['platform'] =~ /windows/
-            "env PATH=\"#{host['privatebindir']}:${PATH}\" cmd /c gem"
+            "env SSL_CERT_FILE=\"C:/Program Files/Puppet Labs/Puppet/puppet/ssl/cert.pem\" PATH=\"#{host['privatebindir']}:${PATH}\" cmd /c gem"
```

Comment:
I think it would fail on the beaker-hostgenerator platform `windows2012r2-6432` (signifying 32 bit puppet MSI installed on 64-bit host). This page https://docs.puppet.com/pe/latest/install_windows.html#what-gets-installed notes that distinction anyway